src/plugins/autoRowSize/test/autoRowSize.e2e.js

```diff
@@ -478,4 +478,19 @@ describe('AutoRowSize', () => {
 
     expect(rowHeight(spec().$container, -1)).toBe(75);
   });
+
+  it('should properly count height', () => {
```

Comment:
I think that this test doesn't cover this bug correctly. When I attached the older version of the handsontable to this test it passes, should fail. Can you check that?